WizardPanels/WizardPanels.js

```diff
@@ -114,14 +119,22 @@ const WizardPanelsBase = kind({
 		 *
  		 * Example:
  		 * ```
-		 * nextButton={<Button icon="closex" aria-label="Quit">Close</Button>}
+		 * nextButton={<Button icon="closex" aria-label="Quit" />}
 		 * ```
 		 *
 		 * @type {Boolean|Component}
 		 * @private
 		 */
 		nextButton: PropTypes.oneOfType([PropTypes.bool, EnactPropTypes.componentOverride]),
 
+		/**
+		 * The label for the next button.
+		 *
+		 * @type {String}
+		 * @private
+		 */
+		nextButtonLabel: PropTypes.string,
```

Comment:
Did you try to make unit test for this?

Reply:
I didn't. 🤔  I will add if new APIs look good to reviewers.

Reply:
APIs are agreed, so I added unit tests.